tests/testthat/examples/Test_2023_1/metadata.yml

```diff
@@ -71,8 +71,7 @@ contexts:
   values:
   - value: female
     description: female plant
-  - find: male
-    value: male
+  - value: male
```

Comment:
Agree this one doesn't make sense. Please ensure there are examples in the tests that do have a reevant example using `find`

Reply:
Yes in the same 'plant sex' `context_property` I put ['female2'](https://github.com/traitecoevo/traits.build/blob/eb4a62ed0efe066744e71af95eb2f4fa172c88ef/tests/testthat/examples/Test_2023_1/metadata.yml#L76) in `find` and 'female' in `value`.